features/admin.branding.v1/components/custom-page-editor/editor-view.tsx

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { CodeEditor, ResourceTab } from "@wso2is/react-components";
+import { Editor } from "codemirror";
+import React, {
+    FunctionComponent,
+    ReactElement,
+    useEffect,
+    useRef
+} from "react";
+
+interface EditorViewTabsProps {
+    html: string;
+    css: string;
+    js: string;
+    readOnly?: boolean;
+    onContentUpdate?: (content: {
+        html: string;
+        css: string;
+        js: string;
+    }) => void;
+    "data-testid"?: string;
+}
+
+export const EditorViewTabs: FunctionComponent<EditorViewTabsProps> = (
+    props: EditorViewTabsProps
+): ReactElement => {
+    const {
+        html,
+        css,
+        js,
+        readOnly,
+        onContentUpdate,
+        "data-testid": testId = "layout-editor-tabs"
+    } = props;
```

Comment:
Let's move this to function signature.

Ref: https://github.com/wso2/identity-apps/blob/66cb2e19b5ba814e6e3dd9cad8fd094728ed237f/features/admin.application-templates.v1/components/application-creation-adapter.tsx#L59-L64